From 00b83b88204b7ea68a9fd7fbd067a48d2c49bad5 Mon Sep 17 00:00:00 2001
From: Juho Kuisma <kuisma.juho+github@gmail.com>
Date: Thu, 28 Dec 2023 11:51:35 +0200
Subject: [PATCH] Patch mocha background

---
 lua/catppuccin/palettes/mocha.lua | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/lua/catppuccin/palettes/mocha.lua b/lua/catppuccin/palettes/mocha.lua
index 296c000..f3996bd 100644
--- a/lua/catppuccin/palettes/mocha.lua
+++ b/lua/catppuccin/palettes/mocha.lua
@@ -22,7 +22,7 @@ return {
 	surface2 = "#585b70",
 	surface1 = "#45475a",
 	surface0 = "#313244",
-	base = "#1e1e2e",
-	mantle = "#181825",
-	crust = "#11111b",
+	base = "#171421",
+	mantle = "#171421",
+	crust = "#171421",
 }
-- 
2.34.1

