From 0f67278cf652f9c1008a862f097f5555be68bf30 Mon Sep 17 00:00:00 2001
From: Juho Kuisma <kuisma.juho+github@gmail.com>
Date: Wed, 20 Nov 2024 18:05:55 +0000
Subject: [PATCH] Patch UTF-8 symbols

---
 autoload/airline/init.vim | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/autoload/airline/init.vim b/autoload/airline/init.vim
index 05ae6ab..a86e070 100644
--- a/autoload/airline/init.vim
+++ b/autoload/airline/init.vim
@@ -148,11 +148,11 @@ function! airline#init#bootstrap()
     call extend(g:airline_symbols, {
           \ 'readonly': "\u229D",
           \ 'whitespace': "\u2632",
-          \ 'maxlinenr': "\u2261",
-          \ 'linenr': " \u33d1:",
-          \ 'colnr': " \u2105:",
-          \ 'branch': "\u16A0",
-          \ 'notexists': "\u0246",
+          \ 'maxlinenr': "",
+          \ 'linenr': " ln:",
+          \ 'colnr': " c:",
+          \ 'branch': "",
+          \ 'notexists': "",
           \ 'crypt': nr2char(0x1F512),
           \ 'dirty': '!',
           \ }, 'keep')
-- 
2.45.2

