From f9982f84bafcbd75e402f8aaddbd3b305a2f08a3 Mon Sep 17 00:00:00 2001
From: Juho Kuisma <kuisma.juho+github@gmail.com>
Date: Tue, 5 Mar 2024 10:48:47 +0200
Subject: [PATCH] Patch UTF-8 symbols

---
 autoload/airline/init.vim | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/autoload/airline/init.vim b/autoload/airline/init.vim
index 9cfd7b0..108c0d8 100644
--- a/autoload/airline/init.vim
+++ b/autoload/airline/init.vim
@@ -148,9 +148,9 @@ function! airline#init#bootstrap()
     call extend(g:airline_symbols, {
           \ 'readonly': "\u229D",
           \ 'whitespace': "\u2632",
-          \ 'maxlinenr': "\u2630",
-          \ 'linenr': " \u33d1:",
-          \ 'colnr': " \u2105:",
+          \ 'maxlinenr': "",
+          \ 'linenr': " ln:",
+          \ 'colnr': " c:",
           \ 'branch': "\u16A0",
           \ 'notexists': "\u0246",
           \ 'crypt': nr2char(0x1F512),
-- 
2.34.1

